docs/DeployingOnCentOS7.md

```diff
@@ -200,31 +226,6 @@ We will be need to set the correct environment variables for each agent and mast
 
 Once these environment variables are set accordingly, we can finally start the Mesos services.
 
-## Mesos Master
-
-### Set Master Environment Variables
-
-Append the following lines to `/etc/default/mesos-master` on each of your master hosts. 
-
-The value of `MESOS_QUORUM` may change depending on how many master hosts you have in your cluster. Mesos recommends that the quorum count is at least 1/2 the number of master hosts running. 
-
-Also note, you will need to fill in the IP address of the host that is running ZooKeeper.
-
-    MESOS_QUORUM=1
-    MESOS_WORK_DIR=/var/lib/mesos
-    MESOS_ZK=zk://<IP of host with zookeeper running>:2181/mesos
-
-If ZooKeeper is not running on your master, you will need to go into `/etc/mesos/zk` and change the line to include the IP address of the host with ZooKeeper running.
-
-### Run Master
-
-Run the mesos-master process on your master host.
-
-    $ sudo systemctl enable mesos-master.service
-    $ sudo systemctl start mesos-master.service
-
-## Mesos Agents
-
 ### Set Agent Environment Variables
```

Comment:
For the Master, the `Set Environment Variables` and `Run` sections are level 2 headings. Once I pull this in I will probably change this sections in the agent to also be level 2 headings (rather than level 3) and move the part about `mesos-init-wrapper -h` into the `Set Environment Variables` section